frontend/src/components/Posts.js

```diff
@@ -10,24 +10,26 @@ const Posts = props => {
   }, []);
 
   const fetchPosts = async () => {
-    const response = await fetch(
-      "http://localhost:5000/posts?_sort=id&_order=DESC"
-    ).catch(error => console.error("Error:", error));
+    const response = await fetch("/api/posts").catch(error =>
+      console.error("Error:", error)
+    );
 
     if (!response) {
       return;
     }
-
     const fetchedPosts = await response.json();
 
-    setPosts(fetchedPosts);
+    // setPosts(fetchedPosts);
```

Comment:
Is this necessary? 🤔

Reply:
this feature will be implemented later 
added TODO as discussed